inc/targetticket.class.php

```diff
@@ -971,6 +973,57 @@ protected function setTargetLocation($data, $formanswer) {
          case self::LOCATION_RULE_SPECIFIC:
             $location = $this->fields['location_question'];
             break;
+         case self::LOCATION_RULE_LAST_ANSWER:
+            $form_id = $formanswer->fields['id'];
```

Comment:
```suggestion
            $formAnswerId = $formanswer->getID();
```

Reply:
Done in 875fd405 + a few other occurrences I found from similar settings.